library/xml

```diff
@@ -382,7 +388,8 @@ def main():
             print_match=dict(required=False, default=None, type='bool'),
             pretty_print=dict(required=False, default=False, type='bool'),
             content=dict(required=False, default=None, choices=['attribute', 'text']),
-            input_type=dict(required=False, default='yaml', choices=['yaml', 'xml'])
+            input_type=dict(required=False, default='yaml', choices=['yaml', 'xml']),
+            strip_tags=dict(required=False, default=True, type='bool')
```

Comment:
Add a comma after this line, so that the next time a parameter gets added, only that change is shown in a diff :-)